### PR TITLE
chore: add regression tests - no ticket

### PR DESCRIPTION
**Pull request by araneforseti**
### Issue

Moving the automation to this repo out of the central QA repo in order to have an easier time keeping the automation updated with the app versions.

### Testing

Change the Jenkins jobs to point to this branch and run the tests

---

### Checklist

- [ ] Title contains a reference JIRA issue number like `[WPB-XXX]`.
- [ ] Description is filled and free of optional paragraphs.
- [ ] Adds/updates automated tests.

---

### UI accessibility checklist

_If your PR includes UI changes, please utilize this checklist:_
- [ ] Make sure you use the API for UI elements that support large fonts.
- [ ] All colors are taken from WireDesign.ColorTheme or constructed using WireDesign.BaseColorPalette.
- [ ] New UI elements have Accessibility strings for VoiceOver.

### CODE REVIEW

**Review by samwyndham — CHANGES_REQUESTED** [review]
One small thing. We should exclude these files from swift formatting and linting. Please update the following files to exclude `wire-ios-automation`:

- `.swiftformat`
- `.swiftlint.yml`

**Comment by araneforseti** [comment]
> One small thing. We should exclude these files from swift formatting and linting. Please update the following files to exclude `wire-ios-automation`:
> 
>     * `.swiftformat`
> 
>     * `.swiftlint.yml`

Should be done now

File: .swiftformat [hunk 1]
```diff
@@ -1,4 +1,4 @@
---exclude **/.build,**/Carthage,**/DerivedData,**/AutoMockable.generated.swift,WireUI/Sources/WireDesign/Icons/Autogenerated/StyleKitIcons.generated.swift
+--exclude **/.build,**/Carthage,**/DerivedData,**/AutoMockable.generated.swift,WireUI/Sources/WireDesign/Icons/Autogenerated/StyleKitIcons.generated.swift,wire-ios-automation
```
Comment (samwyndham) [inline .swiftformat]: I think this should be `wire-ios-automation/**`.

However please first merge `develop` into your branch as `.swiftformat` has changed quite a lot

Comment (araneforseti) [reply .swiftformat]: Yeah you are right. And the new format is far more readable

**Review by samwyndham — APPROVED** [review]
Exciting!

**Comment by araneforseti** [comment]
Closing because of big changes to how we will handle common and will reopen

**Pull request closed without merging**